.travis.yml

```diff
@@ -4,13 +4,16 @@ services:
   - docker
 
 env:
-  - DOCKER_COMPOSE_VERSION=1.16.0

```

Comment:
Cannot we keep this envvat?

Reply:
I haven't found a way to do that, no. Keeping it means a job will be run with only this particular variable set, while missing the other variable (MARATHON_VERSION).